docs/data/material/components/skeleton/YouTube.tsx

```diff
@@ -9,21 +9,21 @@ const data = [
     src: 'https://i.ytimg.com/vi/pLqipJNItIo/hqdefault.jpg?sqp=-oaymwEYCNIBEHZIVfKriqkDCwgBFQAAiEIYAXAB&rs=AOn4CLBkklsyaw9FxDmMKapyBYCn9tbPNQ',
     title: 'Don Diablo @ Tomorrowland Main Stage 2019 | Official…',
     channel: 'Don Diablo',
-    views: '396 k views',
+    views: '396k views',
```

Comment:
```suggestion
    views: '396K views',
```

@samuelsycamore Also, I wonder, shouldn't it be K instead of k?

<img width="505" alt="Screenshot 2022-08-04 at 01 27 21" src="https://user-images.githubusercontent.com/3165635/182729607-a794fd18-d204-4034-8ded-7f572e45e95c.png">

https://www.antidote.info/en/blog/reports/millions-billions-and-other-large-numbers#abbreviated-forms

Reply:
@oliviertassinari I would expect to see lowercase k in US English to refer to units of 1000. Uppercase K would be more common when referring to distance, i.e. running a 10K = 10-kilometer race. In the [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units), lowercase k denotes multiplication by 1000, whereas uppercase K refers to Kelvin (temperature unit).

Reply:
OK thanks, so we are good

*(for the Kelvin, I think it's only relevant for an academic audience, I haven't used the K unit since university, but it's a unit, not a scale)*